src/app/test_executive/zkapps.ml

```diff
@@ -377,7 +377,8 @@ module Make (Inputs : Intf.Test.Inputs_intf) = struct
                     { other_p with
                       authorization =
                         Control.Poly.Proof
-                          (Lazy.force Mina_base.Proof.blockchain_dummy)
+                          (Lazy.force
+                             Mina_base.Proof.For_tests.blockchain_dummy_tag )
```

Comment:
I think we might want to use a different parameter for the db later. For now, it does use a new created value:
```
  let blockchain_dummy_tag =
    Lazy.map
      ~f:(fun dummy -> write_proof_to_disk (For_tests.create_db ()) dummy)
      blockchain_dummy
```